beetsplug/web/__init__.py

```diff
@@ -42,6 +42,10 @@ def _rep(obj, expand=False):
         else:
             del out['path']
 
+        # Filter all bytes attributes and convert them to strings
+        for key in filter(lambda key: isinstance(out[key], bytes), out):
+            out[key] = out[key].decode('utf-8')
```

Comment:
Using `.decode('utf-8')` will assume that the binary data stored on the object happens to be a UTF-8-encoded string. That's not necessarily the case: the acoustid fingerprint, for example, is really a raw binary blob, not any representation of text at all.

Fortunately, there's a [standard library function](https://docs.python.org/2/library/base64.html#base64.b64encode) that can encode bytes as ASCII-safe bytes. Then you can use `.decode('ascii')` to get a text string to pass to the JSON encoder.